src/pages/journal/components/test-slot/test-slot.html

```diff
@@ -1,21 +1,54 @@
-<ion-card>
-  <ion-row align-items-center>
-    <ion-col col-auto>
-      <indicators [showSpecialNeedsIndicator]="isSpecialNeedsSlot()"></indicators>
+<ion-card class="test-slot-card" [ngClass]="{'test-slot-portrait-mode': isPortrait()}">
+  <ion-row align-items-center class="test-slot-row" nowrap>
+    <ion-col col-auto padding-left>
+      <indicators
+        [showSpecialNeedsIndicator]="isSpecialNeedsSlot()"
+      >
+      </indicators>
     </ion-col>
     <ion-col col-2>
-      <time [time]="slot.slotDetail.start" [testComplete]="testComplete"></time>
+      <time
+        [time]="slot.slotDetail.start"
+        [testComplete]="testComplete"
+      >
+    </time>
     </ion-col>
-    <ion-col col-5>
+    <ion-col col-4 style="margin-left: -30px;">
       <candidate
         [name]="slot.booking.candidate.candidateName"
-        [testCategory]="slot.vehicleSlotType"
-        [welshLanguage]="slot.booking.application.welshTest">
+        [welshLanguage]="slot.booking.application.welshTest"
+        [isPortrait]="isPortrait()"
       >
       </candidate>
     </ion-col>
-    <ion-col col-2>
-       <test-details [testCentreName]="slot.testCentre.centreName"></test-details>
+    <ion-col>
+      <ion-grid>
+        <ion-row align-items-center>
+          <ion-col [attr.col-8]="isPortrait() ? true : null" [attr.col-4]="!isPortrait() ? true : null">
+            <test-category-icon
+              [category]="slot.booking.application.testCategory"
+            >
+            </test-category-icon>
+          </ion-col>
+        </ion-row>
+        <ion-row>
+          <ion-col no-padding>
+            <vehicle-details
+              [height]="slot.booking.application.vehicleHeight"
```

Comment:
Might be worth extracting a domain object to encapsulate these vehicle details?

Reply:
I'm going to leave this one for now - there is the potential we are pulling this data out next sprint as part of the Journal-Slot-Rework ticket, I'll add it to the ticket to update it depending on which way we swing.